### ISSUE TITLE
LIf Deposit value

### ISSUE THREAD

**Issue by menschikov** [windingtree/orgid-explorer#39]
is it ok?
![image](https://user-images.githubusercontent.com/4557491/69005773-857bcd80-0948-11ea-95e9-e33900d00556.png)

**Comment by MatiasOS** [comment]
Yes, that is the answer from [backend](https://explorer-cache.myorgid.com/organizations/0xD1cb150B6845FD5704206de5Cf387f05ABD21689) . Do we need to remove some zeroes?

![lif-deposit-siesta](https://user-images.githubusercontent.com/5481398/69655548-8651e500-1055-11ea-8c5e-ee8dc1e8ef85.png)

**Comment by menschikov** [comment]
@kvbik who can fix this on backend side?
